### PR TITLE
VUE-608 KvRadio

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

File: @kiva/kv-components/vue/stories/KvRadio.stories.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	argTypes,
+}) => ({
+	props: Object.keys(argTypes),
+	components: {
+		KvRadio,
+	},
+	data: () => ({
+		radioExampleModel: 'three',
+	}),
+	template: `
```
Comment (eddieferrer) [inline @kiva/kv-components/vue/stories/KvRadio.stories.js:22]: This example seems a little confusing with the model being inside of the label and changing on select. Maybe the example should looks like: 
```
* One radio button
* Two radio button
* Three radio button
* Four radio button

Selected radio value:

Comment (ryan-ludwig) [reply @kiva/kv-components/vue/stories/KvRadio.stories.js:22]: Agreed, thanks!

**Review by eddieferrer — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
Awesome!

**Review by BoulderBrains — COMMENTED** [review]
👀

**Pull request merged**